src/arrayUtils.js

```diff
@@ -0,0 +1,68 @@
+if (typeof maybeExport === 'undefined') {
```

Comment:
Yep, exactly. Sure, I don't mind moving those to jsifier.js. Would that be an improvement, though? It'd save a few bytes for the HTML output, but might also cause someone to forget to keep the `maybeExport`s in sync with the functions in arrayUtils.js.